addons/addon-base-raas/packages/base-raas-cfn-templates/src/templates/onboard-account.cfn.yml

```diff
@@ -28,12 +33,18 @@ Parameters:
     Type: String
     Description: The arn of workflowRunner role
 
-  # Generous subnet allocation of 8192 addresses (ie room for a few concurrent EMR clusters)
-  # ending at 10.0.31.255
-  VpcPublicSubnet1Cidr:
```

Comment:
I think we shouldn't change anything about the account if we don't enable AppStream. Would this try deleting the old subnet or modifying it? Can you double check that existing account remains unchanged.

Reply:
Good point. Updated to keep the CIDR the same for public subnet. 
I checked that the new template keeps the existing account unchanged if AppStream is disabled, and it does.